src/StaticFAQ/FAQExtraInfo/BoardingPassesInfo/BoardingPassesDescription.js

```diff
@@ -31,20 +42,34 @@ const BoardingPassesDescription = ({ data, mmbUrl }: Props) => {
         </p>
         <div className="info">
           {boardingPassUrl ? (
-            <a href={boardingPassUrl} target="_blank" className="download">
-              <Download size="medium" customColor="#00a991" />
-              Download
-            </a>
+            <div
+              onClick={() => downloadBoardingPassTracker()}
```

Comment:
wouldn't be just passing `downloadBoardingPassTracker` enough?

Reply:
also why wrapping by additional div, `<a onClick={...} />` should be IMHO enough

Reply:
double facepalm 🤦‍♂️, don't know what I was thinking 🙏, thanks